contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -1215,8 +1227,13 @@ <h3>Add a Talent to this Contribution</h3>
                                 <div class="form-group">
                                     <label class="control-label col-sm-2">User Privacy Level</label>
                                     <div class="col-sm-10">
-                                        <input max="5" min="1" name="talent_minUserPrivacyLevel"
-                                               step="1" type="number" value="{{ talent|filter_nested_dict(["minUserPrivacyLevel"]) }}"/>
+                                        {%  if is_editable %}
+                                            <input max="5" min="1" name="talent_minUserPrivacyLevel"
```

Comment:
Can you check if there is id for this one in the other PR https://github.com/rokwire/rokwire-building-blocks-api/pull/868. If so, can you add the id in here?